### PR TITLE
Add support for lodash 4

### PR DESCRIPTION
**Pull request by bfaf**
Unit tests pass

### CODE REVIEW

**Comment by bfaf** [comment]
@adleroliveira Please merge or tell me what's wrong so I fix it.

File: dream.js [hunk 1]
```diff
@@ -287,7 +287,12 @@ function Dream() {
     if (isValidSchema(schema)) return schema;
 
     if (typeof (schema) === 'string') {
-      var foundSchema = _.findWhere(_schemas, { name: schema });
+      var foundSchema = null;
```
Comment (paazmaya) [inline dream.js]: Is there a way to do this in a manner that is compatible with both mentioned versions?

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,5 @@ node_modules/
 .idea/
 tsd.json
 typings/
-v2/
\ No newline at end of file
+v2/
+/nbproject/private/
```
Comment (paazmaya) [inline .gitignore]: Is this new line somehow related to the lodash v4 support?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dreamjs",
-  "version": "0.1.7",
+  "version": "0.1.9",
```
Comment (paazmaya) [inline package.json]: Why was version number changed, and jumped over 0.1.8?
Usually the package author does that.

Comment (bfaf) [inline package.json]: Yes, it's author responsibility to change the version but this wasn't done for 0.1.8 so I've decided it to bump it to correct version in this change.

File: .gitignore (hunk 2 above)
Comment (bfaf) [inline .gitignore]: This line slipped my eyes. NetBeans added it without my knowledge.

File: dream.js (hunk 1 above)
Comment (bfaf) [inline dream.js]: I want to keep it backward compatible for a while. It's just a precaution.

**Comment by paazmaya** [comment]
Now nice! 🍻 you get when I see you but only if this gets merged soon...

**Pull request merged**